packages/lit-analyzer/src/lib/analyze/parse/document/text-document/html-document/parse-html-node/parse-html-attr-assignment.ts

```diff
@@ -20,6 +21,19 @@ export function parseHtmlAttrAssignment(
 	const location = getAssignmentLocation(p5Node, p5Attr, htmlAttr, context);
 
 	if (location == null) {
+		// A null assignment location might be an element expression, which only
+		// has an attribute name and no attribute "assignment".
+		if (htmlAttr.name.match(/_+\d+_/)) {
```

Comment:
I guess you'll also need to update this to `/_+[0-9a-z]+_/` to account for the base 10 -> 36 change.

Reply:
oops. I need a test to hit that...

Reply:
done